closure/typedefs/Artifacts.js

```diff
@@ -24,6 +24,15 @@
  * @struct
  * @record
  */
+function NetworkRecord() {}
+
+/** @type {?} */
```

Comment:
maybe add a TODO here? We should be able to grab some externs out of the devtools code pretty easily.